analysers/Analyser_Osmosis.py

```diff
@@ -204,6 +204,34 @@ class Analyser_Osmosis(Analyser):
 CREATE INDEX idx_multipolygons_poly_proj ON {0}.multipolygons USING gist(poly_proj);
 CREATE INDEX idx_multipolygons_tags ON {0}.multipolygons USING gist (tags);
 ANALYZE {0}.multipolygons;
+"""
+
+    sql_create_polygons = """
+CREATE UNLOGGED TABLE {0}.polygons AS
+SELECT
+    'R' AS type,
+    *
+FROM
+    {0}.multipolygons
+UNION ALL
+SELECT
+    'W' AS type,
+    id,
+    tags,
+    CASE WHEN ST_IsValid(linestring) = 't' AND ST_IsSimple(linestring) = 't' AND ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(linestring) ELSE NULL END AS poly,
+    CASE WHEN ST_IsValid(linestring) = 't' AND ST_IsSimple(linestring) = 't' AND ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(ST_Transform(linestring, {1})) ELSE NULL END AS poly_proj,
```

Comment:
```suggestion
    CASE WHEN ST_IsValid(ST_MakePolygon(ST_Transform(linestring, {1}))) THEN ST_MakePolygon(ST_Transform(linestring, {1})) ELSE NULL END AS poly_proj,
```
Same as above, except that here the validity of the poly_proj has to be checked as only the non-projected version is already checked for validity